src/data/flatten-values.rkt

```diff
@@ -151,11 +152,12 @@ becuse it is actually more of a stmt contexts itself.
   (: fv-effect* (D3-Effect* -> D4-Effect*))
   (define (fv-effect* effect*) (map fv-effect effect*))
 
-  (: simplify-assignment (Uid D3-Value -> D4-Effect))
+  (: simplify-assignment ((U Uid String) D3-Value -> D4-Effect))
```

Comment:
If this is a recurring pattern we might want to just name this type L-Value or something.

Reply:
Done.